README.md

```diff
@@ -27,7 +27,26 @@ description: Control the splash screen for your app.
 
 # cordova-plugin-splashscreen
 
-This plugin is required to work with splash screens. This plugin displays and hides a splash screen during application launch.
+This plugin displays and hides a splash screen while your web application is launching. Using its methods you can also show and hide the splash screen manually.
+
+Note that this plugin is not about native loading screens, like e.g. ["Launch Screens" of iOS](https://developer.apple.com/design/human-interface-guidelines/ios/icons-and-images/launch-screen/). This plugin's splash screens are shown _after_ the native app has already started.
```

Comment:
Isn't this only partially true? Android per se does not have any splash screens (at least you have to create a small Activity for this - in contrast Apple forces you to include these launch images)

Reply:
What you write is true, but it doesn't make what I wrote wrong, does it? If you use a splash theme or splash activity on Android, this plugin's splash screen will still only be shown after that is gone. And in general, this plugin only kicks in when the native app has been started.

Reply:
What about just removing this sentence/paragraph again? It might be more confusing than that it helps.

Reply:
Removed for now, created #203 for later.